routes/users.js

```diff
@@ -1,9 +1,62 @@
 var express = require('express');
 var router = express.Router();
+var passport = require('passport');
+const db = require('../models');
+
+//This file needs to run and have nothing to export
+require('../controllers/user.controller');
 
 /* GET users listing. */
 router.get('/', function (req, res, next) {
     res.send('respond with a resource');
 });
 
+router.get('/auth/facebook', passport.authenticate('facebook', { scope : ['email']}));
+
+router.get('/auth/facebook/callback',
```

Comment:
why is this needed? if this is for future requests from Facebook, shouldn't you be using `authorize` instead of `authenticate`?

Reply:
the callback is part of the OAuth mechanism. see my painting drawing for reference